### PR TITLE
jshook/nosqlbench-2031-blipblobs

### LINKED ISSUE #2031: Make modeling occasional 'blip' payloads easy
Some realistic test need to simulate steady state, punctuated by the occasional _big op_. This should be easier.
Binding functions which allow for payload sizes like this could be used for HOF bindings which take a sizing function.

### PR DESCRIPTION
**Pull request by jshook**
closes #2031

### CODE REVIEW

File: nb-virtdata/virtdata-lib-basics/src/test/java/io/nosqlbench/virtdata/library/basics/shared/from_long/to_long/SequenceOfTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package io.nosqlbench.virtdata.library.basics.shared.from_long.to_long;
```
[the remark this answers is not in the document]
Comment (jshook) [reply nb-virtdata/virtdata-lib-basics/src/test/java/io/nosqlbench/virtdata/library/basics/shared/from_long/to_long/SequenceOfTest.java:1]: This got autocorrected to something wrong. I'll update it and avoid imports here completely.

[the remark this answers is not in the document]
Comment (jshook) [reply nb-virtdata/virtdata-lib-basics/src/test/java/io/nosqlbench/virtdata/library/basics/shared/from_long/to_long/SequenceOfTest.java:1]: actually, it was completely missed, fixed in next push

[the remark this answers is not in the document]
Comment (jshook) [reply nb-virtdata/virtdata-lib-basics/src/test/java/io/nosqlbench/virtdata/library/basics/shared/from_long/to_long/SequenceOfTest.java:1]: fixed

**Review by dave2wave — APPROVED** [review]
LGTM

**Pull request merged**